resources/walkthrough/diffs/diff-guide.md

```diff
@@ -0,0 +1,37 @@
+# Comparing Resources
+
+The Confluent extension provides read-only document views for schema definitions and topic message
+previews. You can compare them using VS Code's built-in diff editor:
+
+## Comparing two documents
+
+1. **Select the first document:** Right-click inside a document and select _"Select for Compare"_
+   from the context menu. ![](./topic-message-diff-select.png)
+
+2. **Compare with the second document:** Right-click inside another document and select _"Compare
+   with Selected"_ from the context menu. ![](./topic-message-diff-compare.png)
```

Comment:
These two screenshots look _huge_ in normal previews, but once they're collapsed inside the image/markdown portion of the walkthrough item, they look reasonably-sized